CHANGELOG.md

```diff
@@ -20,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#667](https://github.com/nf-core/mag/pull/667) - Fix pipeline crashing if only CONCOCT selected during binning (reported and fixed by @jfy133)
 - [#670](https://github.com/nf-core/mag/pull/670) - Re-add missing GTDBTk parameters into GTDBTk module (reported by harper357, fixed by @jfy133)
 - [#672](https://github.com/nf-core/mag/pull/673) - Fix GTDB-Tk per-sample TSV files not being published in output directory (reported by @jhayer, fix by @jfy133)
+- Make longread preprocessing a subworkflow
+- Add porechop and filtlong logs to multiqc
+- Change local filtlong module to the official nf-core/filtlong module
 
 ### `Dependencies`
```

Comment:
A table here should be added (copy from previous releases) listing at least the filtlong verison update 0.2.0 -> 0.2.1 and any other tool version updates.

You can also add the versions of the new tools too.